bin/flok-show

```diff
@@ -148,7 +148,7 @@ function rowOf(m, fields) {
 function allMigrations() {
   var m = flok.migrations.slice(0);
   m.sort(function (a, b) {
-    return a.status.runTime - b.status.runTime;
+    return a.status.runTime === b.status.runTime ? 0 : a.status.runTime < b.status.runTime ? -1 : 1;
```

Comment:
what was wrong with the subtraction method?

Reply:
I don't know I feel better when I'm using -1, 0 1 for sorting :)

Reply:
Well technically times will convert to numbers (ms) as you do any math on them, so it's the same thing.  But if it's easier to read them I'm ok with that.